src/EditorFeatures/CSharp/Formatting/CSharpEditorFormattingService.cs

```diff
@@ -29,23 +29,11 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.Formatting
     internal partial class CSharpEditorFormattingService : IEditorFormattingService
     {
         private readonly ImmutableHashSet<char> _supportedChars;
-        private readonly ImmutableHashSet<char> _autoFormattingTriggerChars;
-        private readonly ImmutableDictionary<char, ImmutableHashSet<SyntaxKind>> _multiWordsMap;
 
         public CSharpEditorFormattingService()
         {
-            _autoFormattingTriggerChars = ImmutableHashSet.CreateRange<char>(";}");
-
             // add all auto formatting trigger to supported char
-            _supportedChars = _autoFormattingTriggerChars.Union("{}#nte:)");
-
-            // set up multi words map
-            _multiWordsMap = ImmutableDictionary.CreateRange(new[]
-            {
-                KeyValuePair.Create('n', ImmutableHashSet.Create(SyntaxKind.RegionKeyword, SyntaxKind.EndRegionKeyword)),
-                KeyValuePair.Create('t', ImmutableHashSet.Create(SyntaxKind.SelectKeyword)),
-                KeyValuePair.Create('e', ImmutableHashSet.Create(SyntaxKind.WhereKeyword)),
-            });
+            _supportedChars = ImmutableHashSet.CreateRange<char>(";{}#nte:)");
```

Comment:
I don't see how the immutable hashsets and immutable dictionaries could be a problem, but this immutable hashset won't be a problem.